L1Trigger/TrackerDTC/interface/TTDTCConverter.h

```diff
@@ -31,10 +31,12 @@ GlobalPoint TTDTCConverter(trackerDTC::Settings* settings, const TTDTC::Frame& f
     const std::vector<double>& diskZs = format->diskZs();
     const std::vector<double>& layerRs = format->layerRs();
 
-    const DetId detId = frame.first->getDetId();
+    const DetId& detId = frame.first->getDetId() + settings->offsetDetIdDSV();
```

Comment:
not sure a ref makes sense here; you could do `const DetId detId(frame.first->getDetId() + settings->offsetDetIdDSV());` if you're concerned about avoiding an unnecessary copy